### ISSUE TITLE
visualization broken for intents with quoted material

### ISSUE THREAD

**Issue by jcollas** [RasaHQ/rasa_core#1355]
**Rasa Core version**: 0.12.0

**Python version**: 3.6.5

**Operating system** (windows, osx, ...): os x 10.13.6

**Issue**:

If the story file contains content like so:

* inform{"slot2": "value2"}

Then the html output from running rasa_core.visualizer will not render in a browser.  The html will contain content like

    "/inform{\"slot2\": \"value2\"}"

which won't render.  It needs to be 

    "/inform{\\"slot2\\": \\"value2\\"}"

**Content of domain file** (if used & relevant):
```yaml

```

**Comment by jcollas** [comment]
To reproduce:
```
cd rasa_core/examples/restaurantbot
python -m rasa_core.visualize -s data/babi_stories.md -d restaurant_domain.yml -o graph.html
```
You get an error when opening graph.html in a browser.

**Comment by tmbo** [comment]
Yes that is definitely an issue. Do you mind creating a PR fixing this behaviour? That would be really helpful.

**Comment by jcollas** [comment]
A PR is forthcoming.
